taxi/taxibench_pandas_ibis.py

```diff
@@ -32,70 +26,58 @@ def run_queries(queries, parameters, etl_times):
 
 
 # Queries definitions
-def q1_ibis(table, df_pandas, queries_validation_results, queries_validation_flags, validation):
+def q1_ibis(table, input_for_validation):
     t_query = 0
     t0 = timer()
     q1_output_ibis = (
         table.groupby("cab_type").count().sort_by("cab_type")["cab_type", "count"].execute()
     )
     t_query += timer() - t0
 
-    if validation and not queries_validation_flags["q1"]:
+    if input_for_validation is not None:
```

Comment:
```suggestion
    if input_for_validation:
```
this is shorter and conveys meaning better. besides it works as a little of a safety net for passing a `{}` inside